### PR TITLE
#696 Add github pages template

### PR DESCRIPTION
**Pull request by Zejnilovic**
Thanks to @GeorgiChochov and @yruslan for pointing in me in the right
direction.

I invited for review even people "outside" of Enceladus project because we will want to use similar template for everyone.

### CODE REVIEW

File: _data/versions.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- '1.0.0'
+- '1.1.0'
```
Comment (GeorgiChochov) [inline _data/versions.yaml]: remove this

File: _includes/navigation.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      <a href="https://mvnrepository.com/artifact/za.co.absa" target="_blank" class="text-dark">
+        <i class="fas fa-file-archive fa-2x"></i>
+
+        <p>Maven</p>
+      </a>
+    </li>
+  </ul>
+</nav>
+
+<!--  p-3 mb-2 bg-secondary text-white
```
Comment (GeorgiChochov) [inline _includes/navigation.html]: Remove?

File: _includes/navigation.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  <h3>Navigation</h3>
+
+  {% if page.version %}
+    {% assign v = page.version %}
+  {% else %}
+    {% assign v = site.data.versions.last %}
+  {% endif %}
+
+  <ul class="nav flex-column nav-pills nav-fill">
+    <!-- <li class="nav-item "><a class="nav-link" href="https://github.com/AbsaOSS">AbsaOss</a></li> -->
```
Comment (GeorgiChochov) [inline _includes/navigation.html]: Remove?

File: _includes/head.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta lang="en">
+  <meta name="generator" content="Jekyll v{{ jekyll.version }}">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/149f23b8d9.js"></script>
+  <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/styles.css">
```
Comment (leborgneromain) [inline _includes/head.html]: I think it should be styles.scss instead of styles.css

File: _sass/main.scss [hunk 5]
```diff
@@ -0,0 +1,39 @@
+html, body {
+  height: 100%;
+}
+
+header {
+  padding-bottom: 1rem;
+}
+
+/* Set black background color, white text and some padding */
```
Comment (leborgneromain) [inline _sass/main.scss]: Unecessary comment (Especially because this is not black but more grey 😆)

File: assets/js/common.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+---
+
+function switchVersion(selectedOption, urlPostfix) {
+  window.location.href = "{{ site.url }}{{ site.baseurl }}/docs/" + selectedOption + "/" + urlPostfix;
```
Comment (leborgneromain) [inline assets/js/common.js]: From the CQC meetings, prefer the string interpolation.

File: _includes/head.html (hunk 4 above)
Comment (Zejnilovic) [reply _includes/head.html]: The Jekyll generates `styles.css` even though definitions are in `scss`
image

Comment (leborgneromain) [reply _includes/head.html]: Ah ok.

**Comment by GeorgiChochov** [comment]
Actually, can you update the `.gitignore` to cover our untracked files from other branches? when switching branches they hang around and can be easily committed by mistake here.

**Comment by Zejnilovic** [comment]
Will be merged on Monday to give enough space to everyone.

**Pull request merged**